pyirf/tests/test_utils.py

```diff
@@ -49,3 +49,59 @@ def test_check_table():
 
     # m is convertible
     check_table(t, required_units={'bar': u.cm})
+
+    t = Table({'bar': [0, 1, 2]})
+    with pytest.raises(WrongColumnUnit):
```

Comment:
Since you explicitly defined the error message in the exceptions module I think you can be explicit also in its testing,

`with pytest.raises(WrongColumnUnit, match=expected_error_message)`

same for `test_calculate_theta`

Reply:
Done